README.md

```diff
@@ -72,3 +72,56 @@ it still requires PostgreSQL.
 
 `mock_siri_server.py` will make up random arrival times for random routes when queried, but all the routes would be valid ones that actually
 stop on the requested stop according to the GTFS database. Make sure to run `update_feed.sh` before running the mock server.
+
+
+## Deploy with Docker
+1. Create a file ``docker-compose.yml``:
+
+```
+version: '3.1'
+  
+services:
+  curlbus:
+    container_name: curlbus
+    image:  guysoft/curlbus
+    volumes:
+      - ./config.ini:/curlbus/config.ini:ro
+    tty: true
+    links:
+      - "db:postgres"
+    ports:
+      - 8080:80
+
+  db:
+    image: postgres
+    restart: always
+    container_name: curlbus-db
+    environment:
+      POSTGRES_PASSWORD: example
+      POSTGRES_DB: curlbus
+    volumes:
+      - curlbus-data:/var/lib/postgresql/data
+
+volumes:
+  curlbus-data:
+```
+2. 
+```
+wget https://raw.githubusercontent.com/guysoft/curlbus/master/config.ini.docker -O config.ini
```

Comment:
Probably should wget this from my account, otherwise it's a bit weird because documentation will be maintained here and configuration in your repo?

Reply:
It was not there, will update but you will only work after you pull

Reply:
Also, you might want to create a docker hub automated build so ``image:  guysoft/curlbus`` could be updated